### DISCUSSION TITLE
Beta 12 with Traefik reverse proxy. go2rtc keeps rejecting the Origin.

### DISCUSSION THREAD

**Discussion by mossyhub** [blakeblackshear/frigate#5196] [Ask A Question] [answered]
I have beta 12 running and any time I access Frigate externally ( or even by my domain name internally) webRTC streams fail with the below error. I see lots of post online when using gorilla websockets but i cannot seem to get it to work. Anyone else having this issue?

I am using Traefik with my own domain name.

23:15:12.345 ERR github.com/AlexxIT/go2rtc/cmd/api/ws.go:66 > host=192.168.0.9 origin=https://cams.mydomain.com error="websocket: request origin not allowed by Upgrader.CheckOrigin"

**Comment by NickM-27** [comment, marked as answer]
Recommend reading the go2rtc docs, this is covered there

**Reply by mossyhub** [reply]
my bad. For some reason it didn't click that api: origin: "*". My brain read it a hundred times as the web ui allowing it. Ugh.
